Gemfile.lock

```diff
@@ -221,7 +221,7 @@ GEM
       activemodel (>= 4.2, < 7.0)
       activerecord (>= 4.2, < 7.0)
       request_store (~> 1.0)
-    govspeak (6.5.10)
+    govspeak (6.6.0)
```

Comment:
Might be better to make this version an explicit dependency in `Gemfile` too. Otherwise another dependency could in theory require an older version of govspeak, and there'd be no compatibility clash, so Gemfile.lock would merrily downgrade and then the `<details>` element would break again.

Reply:
Ooh, interesting! Yes I think our reasoning still holds up and is worth applying here, where we know that a lower version would break Whitehall.

Reply:
I wouldn't expect it's worth the long term overhead this will bring. I assume the test would fail if the version was lowered?

Putting a pin is advised more for when you need to stay behind the latest version.

Reply:
the tests will fail, and following the docs and this advice I've removed the constraint once more 👍